actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/OrgBulkUploadActor.java

```diff
@@ -0,0 +1,100 @@
+package org.sunbird.learner.actors.bulkupload;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.*;
+import org.sunbird.common.request.ExecutionContext;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.learner.actors.bulkupload.model.BulkUploadProcess;
+import org.sunbird.learner.util.Util;
+
+/** Created by rajatgupta on 15/10/18. */
+@ActorConfig(
+  tasks = {"orgBulkUpload"},
+  asyncTasks = {}
+)
+public class OrgBulkUploadActor extends BaseBulkUploadActor {
+
+  private String[] bulkOrgAllowedFields = {
+    JsonKey.ORGANISATION_NAME,
+    JsonKey.CHANNEL,
+    JsonKey.IS_ROOT_ORG,
+    JsonKey.PROVIDER,
+    JsonKey.EXTERNAL_ID,
+    JsonKey.DESCRIPTION,
+    JsonKey.HOME_URL,
+    JsonKey.ORG_CODE,
+    JsonKey.ORG_TYPE,
+    JsonKey.PREFERRED_LANGUAGE,
+    JsonKey.THEME,
+    JsonKey.CONTACT_DETAILS,
+    JsonKey.LOC_ID,
+    JsonKey.HASHTAGID,
+    JsonKey.LOCATION_CODE
+  };
+
+  @Override
+  public void onReceive(Request request) throws Throwable {
+    Util.initializeContext(request, TelemetryEnvKey.GEO_LOCATION);
+    ExecutionContext.setRequestId(request.getRequestId());
+    String operation = request.getOperation();
+
+    switch (operation) {
+      case "orgBulkUpload":
+        upload(request);
+        break;
+      default:
+        onReceiveUnsupportedOperation("OrgBulkUploadActor");
+    }
+  }
+
+  private void upload(Request request) throws IOException {
```

Comment:
Push common code to base class and reuse in LocationBulkUploadActor and OrgBulkUploadActor.

handleUpload(request, type, allowedFields)